src/index.js

```diff
@@ -168,7 +168,7 @@ export default (function create(/** @type {Options} */ defaults) {
 			data = f(data, options.headers) || data;
 		});
 
-		if (data && typeof data === 'object' && typeof data.append !== 'function') {
+		if (data && typeof data === 'object' && typeof data.append !== 'function' && !(data instanceof Blob)) {
```

Comment:
Thinking this might be smaller:

```suggestion
		if (data && typeof data === 'object' && /^\[object (FormData|Blob)\]$/.test(data)) {
```